kuma/wiki/tests/test_views.py

```diff
@@ -435,31 +434,28 @@ def test_disabled_rendering(self, mock_kumascript_get):
         in rendering"""
         mock_kumascript_get.return_value = (self.doc.html, None)
         self.doc.schedule_rendering('max-age=0')
-        ok_(not mock_kumascript_get.called,
-            "kumascript not should have been used")
+        assert not mock_kumascript_get.called, "kumascript not should have been used"
```

Comment:
Again, the original wording of the error message is really awkward. We should improve it with `kumascript should not have been used` instead.